packages/frontend/amp/pages/Article.tsx

```diff
@@ -9,6 +9,7 @@ import { AdConsent } from '@frontend/amp/components/AdConsent';
 import { css } from 'emotion';
 import { Sidebar } from '@frontend/amp/components/Sidebar';
 import { Analytics, AnalyticsModel } from '@frontend/amp/components/Analytics';
+import { filterForTagsOfType } from '../lib/tag-utils';
```

Comment:
Minor, but we tend to use absolute import paths - @frontend/... 

We should probably find a way to lint this though! As I think everyone does this.

Reply:
Sounds good. Will change all as way easier to read as a dev too! Perhaps there is a linter plugin already written?

Looks like there is a TSlint rule that already exists for `no-relative-imports` that I'll look into: https://github.com/Microsoft/tslint-microsoft-contrib/issues/435

Reply:
thanks - sounds great! :)